cmd/utils/dev.go

```diff
@@ -212,98 +202,10 @@ func GetDevFromManifest(manifest *model.Manifest, devName string) (*model.Dev, e
 		return nil, err
 	}
 
-	return manifest.Dev[devKey], nil
-}
-
-//AskYesNo prompts for yes/no confirmation
-func AskYesNo(q string) (bool, error) {
-	var answer string
-	for {
-		oktetoLog.Question(q)
-		if _, err := fmt.Scanln(&answer); err != nil {
-			return false, err
-		}
-
-		if answer == "y" || answer == "n" {
-			break
-		}
-
-		oktetoLog.Fail("input must be 'y' or 'n'")
-	}
-
-	return answer == "y", nil
-}
-
-func AskForOptions(options []string, label string) (string, error) {
-	selectedTemplate := `{{ " ✓ " | bgGreen | black }} {{ .Label | green }}`
-	activeTemplate := fmt.Sprintf("%s {{ . | oktetoblue }}", promptui.IconSelect)
-	inactiveTemplate := "  {{ . | oktetoblue }}"
-	if runtime.GOOS == "windows" {
-		selectedTemplate = " ✓  {{ . | blue }}"
-		activeTemplate = fmt.Sprintf("%s {{ . | blue }}", promptui.IconSelect)
-		inactiveTemplate = "  {{ . | blue }}"
-	}
-
-	prompt := promptui.Select{
-		Label: label,
-		Items: options,
-		Size:  len(options),
-		Templates: &promptui.SelectTemplates{
-			Label:    "{{ . }}",
-			Selected: selectedTemplate,
-			Active:   activeTemplate,
-			Inactive: inactiveTemplate,
-			FuncMap:  promptui.FuncMap,
-		},
-	}
-	prompt.Templates.FuncMap["oktetoblue"] = oktetoLog.BlueString
-
-	i, _, err := prompt.Run()
-	if err != nil {
-		oktetoLog.Infof("invalid init option: %s", err)
-		return "", fmt.Errorf("invalid option")
-	}
-
-	return options[i], nil
-}
-
-//AskIfOktetoInit asks if okteto init should be executed
-func AskIfOktetoInit(devPath string) bool {
```

Comment:
This was only used at up.go, i moved the logic as AskYesNo from there and removed this function